b3d/utils.py

```diff
@@ -103,7 +103,7 @@ def pad_with_1(x):
 
 
 def make_mesh_from_point_cloud_and_resolution(grid_centers, grid_colors, resolutions):
-    box_mesh = trimesh.creation.box(jnp.ones(3))
+    box_mesh = trimesh.load(os.path.join(b3d.get_assets_path(), "objs/cube.obj"))    
```

Comment:
@nishadgothoskar is there a reason for this change?

Reply:
(I merged this in any case since I don't want us to get slowed down -- but a bit curious what the reason was here!)

Reply:
yeah! the reason here was that when you share vertices between faces that are actually on two different faces (of the 6 faces of a cube), then doing shading based on lighting to adjust the vertex colors is problematic